### ISSUE TITLE
Module not found when using docker

### ISSUE THREAD

**Issue by yanlong-pan** [mmabrouk/chatgpt-wrapper#276]
After pulling down the repo and running `docker-compose build && docker-compose up -d`, the container is started with the following error message in the logs.

```
Traceback (most recent call last):
File "chatgpt_wrapper/gpt_api.py", line 5, in <module>
from chatgpt_wrapper.backends.openai.api import OpenAIAPI
ModuleNotFoundError: No module named ‘chatgpt_wrapper'
```

**Comment by thehunmonkgroup** [comment]
Docker support is experimental, and was likely broken by recent updates.

You're welcome to sponsor this work, and/or submit a PR to fix the issue -- it's not something I'm likely to work on personally.

**Comment by thehunmonkgroup** [comment]
See this PR, which will hopefully go in soon as a fix: #281
